qiskit_experiments/library/hamiltonian/heat_zx.py

```diff
@@ -0,0 +1,220 @@
+                 └─────────┘ ░ └───────┘└────────────────┘└───────┘ ░ └───┘└╥┘
+            c: 1/═══════════════════════════════════════════════════════════╩═
+                                                                            0
+
+                             (xN)
+                    ┌───┐    ░ ┌───────┐                       ░
+            q_0: ───┤ X ├────░─┤0      ├───────────────────────░─────────
+                 ┌──┴───┴──┐ ░ │  heat │┌───────────┐┌───────┐ ░ ┌───┐┌─┐
+            q_1: ┤ Rα(π/2) ├─░─┤1      ├┤ Rx(angle) ├┤ Rβ(π) ├─░─┤ γ ├┤M├
+                 └─────────┘ ░ └───────┘└───────────┘└───────┘ ░ └───┘└╥┘
+            c: 1/══════════════════════════════════════════════════════╩═
+                                                                       0
+
+        ZX-HEAT experiments are performed with combination of two
+        error amplification experiments shown above, where :math:`\\alpha, \\beta, \\gamma`
+        depend on the interrogated error axis, namely,
+        (``X``, ``X``, ``I``), (``Y``, ``Y``, ``I``), (``Y``, ``Z``, ``Rx(-pi/2)``)
+        for amplifying X, Y, Z axis, respectively.
+        The circuit in middle is repeated by ``N`` times for the error amplification.
+
+    # section: note
+        The ``heat`` gate is a special gate to represent the entangler pulse sequence.
+        This gate is usually not provided by the backend, and thus user must provide
+        the pulse definition to run this experiment.
+        This pulse sequence should be pre-calibrated to roughly implement the
+        ZX(angle) evolution otherwise selective amplification doesn't work properly.
+
+    # section: see_also
+        qiskit_experiments.library.hamiltonian.HeatElement
+
+    # section: analysis_ref
+        :py:class:`HeatAnalysis`
+
+    # section: reference
+        .. ref_arxiv:: 1 2007.02925
+    """
+
+    def __init__(
+        self,
+        qubits: Tuple[int, int],
+        error_axis: str,
+        backend: Optional[Backend] = None,
+        angle: Optional[float] = np.pi / 2,
+    ):
+        """Create new HEAT experiment for the entangler of ZX generator.
+
+        Args:
+            qubits: Index of control and target qubit, respectively.
+            error_axis: String representation of axis that amplifies the error,
+                either one of "x", "y", "z".
+            backend: Optional, the backend to run the experiment on.
+            angle: Angle of controlled rotation, which defaults to pi/2.
+
+        Raises:
+            ValueError: When ``error_axis`` is not one of "x", "y", "z".
+        """
+
+        amplification_exps = []
+        for control in (0, 1):
+            prep = QuantumCircuit(2)
+            echo = QuantumCircuit(2)
+            meas = QuantumCircuit(2)
+
+            if control:
+                prep.x(0)
+                echo.rx(angle, 1)
+            else:
+                echo.rx(-angle, 1)
```

Comment:
The minus sign does not seem to be reflected in the circuits in the doc.

Reply:
This is reflected. There are two circuits in the parsed literal

Reply:
Right, I was looking at the wrong gate.